dc-cudami-editor/src/components/WebpageForm.jsx

```diff
@@ -35,47 +42,43 @@ const WebpageForm = (props) => {
       </Row>
       <Row>
         <Col sm="12">
-          {props.identifiable.uuid && (
-            <FormIdInput id={props.identifiable.uuid} />
-          )}
+          {identifiable.uuid && <FormIdInput id={identifiable.uuid} />}
           <Nav tabs>
-            {props.existingLanguages.map((language) => (
+            {existingLanguages.map((language) => (
               <LanguageTab
-                activeLanguage={props.activeLanguage}
+                activeLanguage={activeLanguage}
                 key={language}
                 language={language}
-                onClick={(language) => props.onToggleLanguage(language)}
+                onClick={(language) => onToggleLanguage(language)}
```

Comment:
Kann man hier nicht `onToggleLanguage` direkt übergeben? `this`-gebunden müsste es ja vom Parent sein, gibt also eigentlich keinen Grund hier ne Arrow-Function drumzumachen, oder überseh ich was?

Reply:
Vollkommen richtig, klappt auch wunderbar ohne diese Verrenkung :+1: